projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/subscribers.php

```diff
@@ -72,6 +84,22 @@ public function get_subscriber_count( $request ) { // phpcs:ignore VariableAnaly
 			'count' => $subscriber_count,
 		);
 	}
+
+	/**
+	 * Retrieves splitted subscriber counts
+	 *
+	 * @return array data object containing subscriber counts ['email_subscribers' => 0, 'social_followers' => 0]
+	 */
+	public function get_subscriber_counts() {
+		if ( ! Constants::is_defined( 'TESTING_IN_JETPACK' ) ) {
+			delete_transient( 'wpcom_subscribers_totals' );
+		}
+
+		$subscriber_info  = Jetpack_Subscriptions_Widget::fetch_subscriber_counts();
+		$subscriber_count = $subscriber_info['value'];
```

Comment:
Would it make sense to use plural here too? As in `subscriber_counts`? It is about semantics only.